tests/Integration/Behaviour/Features/Context/Domain/Product/ProductImageFeatureContext.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Integration\Behaviour\Features\Context\Domain\Product;
+
+use PrestaShop\PrestaShop\Core\Domain\Product\Command\AddProductImageCommand;
+use RuntimeException;
+
+class ProductImageFeatureContext extends AbstractProductFeatureContext
+{
+    /**
+     * @When I add new product :productReference image :fileName
+     *
+     * @param string $productReference
+     * @param string $fileName
+     */
+    public function uploadImage(string $productReference, string $fileName)
+    {
+        //@todo: behats database contains empty ImageType.
```

Comment:
Well isn't it a bug in the fixtures? I would expect ImageType to always be present in DB

Reply:
Its something related to this https://github.com/PrestaShop/PrestaShop/issues/17725

Reply:
actually the PR is merged, types exists and I've already added some tests for that, so this todo can be removed